variables.tf

```diff
@@ -115,3 +115,9 @@ variable "selections" {
   type        = list
   default     = []
 }
+
+variable "enabled" {
+  description = "Whether or not deploy anything"
```

Comment:
I would like another description, something like:

variable "enabled" {
  description = "Change to `false`  to avoid deploying any AWS Backup resources"
  type        = bool
  default     = true
}